SimplyBadCode.cls

```diff
@@ -0,0 +1,8 @@
+class SimplyBadCode {
+  public static void loopExample(){
+    for(Integer i = 0; i <500; i++){
+      Contact c = new Contact(name='name '+i);
+      insert c;
```

Comment:
R: When we do DML in a for loop, the platform may trigger Governor limits. <<Link to governor limits>> 

We expect code in our codebase to persist lists of information outside of the for loop. 

```apex
List<Contact> contacts = new List<Contact>();
for(Integer i=0; i<500; i++){
....
}
insert contacts;
```